src/test/java/com/github/tomakehurst/wiremock/SnapshotDslAcceptanceTest.java

```diff
@@ -120,7 +119,7 @@ void snapshotRecordsAllLoggedRequestsWhenNoParametersPassed() throws Exception {
   }
 
   @Test
-  void supportsFilteringByCriteria() {
+  public void supportsFilteringByCriteria() throws Exception {
```

Comment:
Not sure about this - adding `public` and `throws Exception` where not needed just adds noise IMO.

Reply:
I feel more comfortable when IDEA no show warning)))